realtor/test/test_users.py

```diff
@@ -0,0 +1,224 @@
+import json
+import uuid
+
+import pytest
+from users.models import User
+from users.serializers import UserSerializer
+
+
+@pytest.fixture
+def test_password():
+    return "strong-test-pass"
+
+
+@pytest.fixture
+def test_phone():
+    return "01000000000"
+
+
+@pytest.fixture
+def test_date():
+    return "2023-05-08T22:55:10.837518Z"
```

Comment:
1. 가능하면 fixture의 이름에 `test_` prefix나 `_test` suffix를 사용하지 않는 것이 좋습니다. 기계는 데코레이터를 보고 피하지만, 사람은 피하기 어려울 수 있습니다.
2. 하나의 fixture로 합칠 수 있어 보이네요.

```suggestion
@pytest.fixture
def auth_info_preset():
    return {
        "password": "strong-test-pass",
        "phone": "01000000000",
        "date_joined": "2023-05-08T22:55:10.837518Z",
    }
```